tests/codegen/queries/optional_list_of_people.graphql

```diff
@@ -0,0 +1,6 @@
+query OptionalListOfPeople {
```

Comment:
IMHO, it would be beneficial to use an operation name here that differs from the name of the queried field to make it easier to verify the generated code is correct. We could go with just `OperationName`.